app/views/devise/sessions/new.html.erb

```diff
@@ -15,6 +15,8 @@
       </div>
     <% end -%>
 
+    <%= f.password_field :password: false, 'pw button': t('devise.showPassword') %>
```

Comment:
We'll want to replace this with a React component entirely, something like:
```
<%= react_component('InputPassword', props: {}) %>
```

Reply:
Here is our [docs](https://github.com/ifmeorg/ifme/wiki/Frontend-Practices#how-to-use-react-in-rails) on how to use React in Rails.